docs/howto/create-a-release.rst

```diff
@@ -102,6 +102,115 @@ Upload build to PyPI
+(N.B. conda packages do not use PYPI for dependencies, they all use
+other conda packages as dependencies!.
+- Check if you need to update the version in the Jinja2 variable right
+at the top of the file.
+- If you updated the version, set the build number back to 0.
+- If you did not update the version, increase the build number.
+- If the conda package contains a source URL, check that it is evaluated
+correctly after the version is replaced, and update the ``sha256`` value.
+- Check requirements (we may have changed setup.py, or package.json, etc.
+ensure we are up to date).
+- Ensure the test commands are still valid
+- Check if you need to change summary, description, or any other field.
+- Create a branch in your clone, commit, and push to your fork. Wait
+for the feedback of the Conda Forge CI bots.
+- In the meantime, test the package locally (see below).
+- If the bots gave you the OK, and testing locally worked fine too,
+your pull request should be ready for review.
+- If you changed something in the recipe, and you are not sure if
+that will affect the Conda Forge package, you can ask Conda Forge
+maintainers help reviewing the pull request (via Gitter).
+- Otherwise, get others from Cylc Core team to review & approve. Once
+your pull request has been approved and merged, the bots will start the
+release process.
+
+From past experience, I've seen Conda packages available in Conda Forge
+in under 30 minutes, but have also had to wait hours (<8 hours) for it
+to show in Conda Forge. For example, if Azure or GitHub infrastructure
+have issues (as I experienced during the first releases) it may take days.
+So keep that in mind before announcing the Conda packages.
+
+To test packages locally, first you should make sure that Conda is
+configured to avoid automatically upload the package. Open you ``~/.condarc``
+and check that you have something similar to:
+
+.. code-block:: yaml
+    channels:
+      - defaults
+      - conda-forge
+    ssl_verify: true
+    anaconda_upload: false
+
+Now create a Conda environment for your tests, e.g.: ``conda create -n cylc1``,
+and then activate it ``conda activate cylc1``. Then to build and install
+locally:
+
+.. code-block:: bash
+    # Where $CONDA_FORGE_REPOSITORY could be, for example,
+    # cylc-uiserver-feedstock.
+    cd $CONDA_FORGE_REPOSITORY
+
+    # Your package should not be listed!
+    conda list
+
+    # This will take some minutes and print useful information.
+    conda build recipe/
+
+    # The following command will install the locally created package. Before
+    # installing it will ask you to confirm. Scroll up and search the
+    # package name. The right-side column must show a location like
+    # .../anaconda3/conda-bld/linux-64::cylc-uiserver-0.1-py37_1.
+    # This confirms you are installing the local build. Here $PACKAGE_NAME
+    # could be something like cylc-uiserver.
+    conda install $PACKAGE_NAME
```

Comment:
```suggestion
.. code-block:: bash

   # Where $CONDA_FORGE_REPOSITORY could be, for example,
   # cylc-uiserver-feedstock.
   cd $CONDA_FORGE_REPOSITORY
   # Your package should not be listed!
   conda list
   # This will take some minutes and print useful information.
   conda build recipe/
   # The following command will install the locally created package. Before
   # installing it will ask you to confirm. Scroll up and search the
   # package name. The right-side column must show a location like
   # .../anaconda3/conda-bld/linux-64::cylc-uiserver-0.1-py37_1.
   # This confirms you are installing the local build. Here $PACKAGE_NAME
   # could be something like cylc-uiserver.
   conda install $PACKAGE_NAME
```